feat: blog: Using AST to build rules in ESLint (#324)
eslint/eslint.org: src/_includes/partials/author_bios/gayathri-krishnaswamy.md

> I am a freelance technical writer who is passionate about improving the documentation of open-source projects.I enjoy solving documentation challenges holistically with people and technology.  When I am not writing, I enjoy cooking Indian food and spending time with my lovely girls.

Reviewer: ```suggestion
I am a freelance technical writer who is passionate about improving the documentation of open-source projects. I enjoy solving documentation challenges holistically with people and technology.  When I am not writing, I enjoy cooking Indian food and spending time with my lovely girls.
```

Reviewer 2: Making a bit more formal here.
```suggestion
Gayathri Krishnaswamy is a freelance technical writer who is passionate about improving the documentation of open-source projects. She enjoys solving documentation challenges holistically with people and technology. When she isn't writing, Gayathri enjoys cooking Indian food and spending time with her lovely girls.
```

Author: Resolved.
